experimental/packages/opentelemetry-instrumentation-http/src/http.ts

```diff
@@ -717,6 +713,11 @@ export class HttpInstrumentation extends InstrumentationBase<Http> {
       code: utils.parseResponseStatus(SpanKind.SERVER, response.statusCode),
     });
 
+    const route = attributes[SemanticAttributes.HTTP_ROUTE];
+    if (route) {
+      span.updateName(`${request.method || 'GET'} ${route}`);
+    }
```

Comment:
Maybe an entry for this new feature in the changelog as well?

Reply:
done